tests/network-tests/src/fixtures/council/ElectCouncilFixture.ts

```diff
@@ -37,7 +36,10 @@ export class ElectCouncilFixture extends BaseQueryNodeFixture {
     await new FixtureRunner(addStakingAccountsFixture).run()
 
     const votersStakingAccounts = (await this.api.createKeyPairs(numberOfVoters)).map(({ key }) => key.address)
-    await api.treasuryTransferBalanceToAccounts(votersStakingAccounts, voteStake.addn(MINIMUM_STAKING_ACCOUNT_BALANCE))
+    await api.treasuryTransferBalanceToAccounts(
+      votersStakingAccounts,
+      voteStake.add(api.consts.referendum.minimumStake)
```

Comment:
`voteStake` is already `=== api.consts.referendum.minimumStake`
I think the `MINIMUM_STAKING_ACCOUNT_BALANCE` addition was just a wrong assumption I made at some point about locks not "stacking" on top of each other. I tried changing this line to:
```suggestion
      voteStake
```
And the test is still passing